samples/DataImporter/GraphEngine.DataImporter/DsvImporter.cs

```diff
@@ -85,7 +92,7 @@ private static List<string> CsvSplit(string line)
             for (; curIndex < line.Length; curIndex++)
             {
                 char c = line[curIndex];
-                if ((c == ',' || c == '\t') && cDoubleQuotes % 2 == 0)
+                if (separators.Contains(c) && cDoubleQuotes % 2 == 0)
                 {
                     fields.Add(SanitizeCsvField(line.Substring(beginIndex, curIndex - beginIndex)));
                     beginIndex = curIndex + 1;
```

Comment:
I also have some questions on the quotes processing. But lets focus on the delimiter processing in this PR.